examples/stocks/test_driver/scroll_perf.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_driver/driver_extension.dart';
+import 'package:flutter_driver/src/error.dart';
+import 'package:stocks/main.dart' as app;
+
+void main() {
```

Comment:
Currently we key off the `_test.dart` suffix, so putting this `_perf.dart` file in `test` should be fine.  We should have TestOn support pretty soon.  The test package just published a release that will let us integrate in a less hacky way.

Anyway, I didn't mean to bikeshed your patch.  We can always move the file around.  :)